ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+décision ouverte chez Red Hat ?
+</td>
+</tr>
+</table>
+
+<br/>
+
+Resources <a name="resources"></a>
+===========================================================================
+
+_page 13 of 15_ 
+
+  - [Red Hat Multiplier](https://github.com/red-hat-people-team/red-hat-multiplier) –
+Pense-bête sur la collaboration, la transparence, la confiance, la méritocracy
+et la connection 
+  - [The Open Source Way handbook](https://www.theopensourceway.org/wiki/Introduction) –  guide pour
+créer et entretenir une communauté de crontributeurs
+  - [The Open Organization](http://opensource.com/open-organization) (livre +
+    communauté en ligne)
+  - [Prioritizing by impact](http://developerblog.redhat.com/2014/08/27/5-ux-tips-for-developers/),
+voir le tableau de Máirín Duffy : "5 UX Tips for Developers"
+  - [Opensource.com](http://www.opensource.com/) – Un publication soutenue par
+    Red Hat focalisée sur comment les principes open source peuvent être
+appliqués aux entrerpises, à l'éducation, aux gouvernements et ailleurs
+  - [The Advice Process](http://danieltenner.com/2014/11/06/the-advice-process-definition-and-usage-tips/) (Daniel Tenner)
+
+<br/>
+
+Annexes <a name="appendix"></a>
+===========================================================================
+
+Histoire: D'où vient le Cadre de Décision Ouverte <a name="history"></a>
+----------------------------------------------------
+
+_page 14 sur 15_ 
+
+  - Basé sur des principes practiqués par des communautés open source
+
+    - Recheches par la Fuqua School of Business à l'université de Duke et Diana
+      Martin (2009 – 2010); ressources communautaire supplémentaires
+
+  - Developpé par l'équipe Red Hat People avec des contributions de groupes
+    témoins inter-fonctionnels  
+
+    - Issu de l'effort "Project Management Office" pour créer un méthodologie de
+      gestion de projet ouvert. (2012 – 2013)
+    - Une conversation via une liste mémo d'un calendrier Google utilisée comme
+      catalysateur pour partager des brouillons avec tous les collaborateurs 
+      et inviter à la participation (2014)
+    - Testé par les équipes IT et Engineering, dans le groupe de travail basé
+      sur Google Calendar (2014 – 2015)
+
+  - Mis à jour et maintenu par Rebecca Fernandez (rfernand@redhat.com)
+
+<br/>
+
+Pourquoi ce cadre existe <a name="why-framework"></a>
+------------------------
+
+_page 15 sur 15_ 
+
+**Un ensemble de partiques éprouvées qui :**
+
+  - Conduit à un meilleur alignement entre les décisions d'entreprise et la
+    stratégie, les buts, la culture, les valeurs et la mission de notre société
+  - Démontre “A quoi ressemble les bonnes pratiques" en terme de prise de
+    décision et de communication
+  - Offre un guide cohérent pour les équipes et les leaders à propos des
+    attentes culturelles de Red Hat, équilibrant la transparence et la confidentialité
+  - Améliorer l'implication des collaborateurs, le ratio signal-bruit sur les
+    listes-memo
```

Comment:
I am not sure we need to translate "memo-list", as that's the name of the mailling list, (more info on http://www.managementexchange.com/story/employee-led-evolution-memo-list-red-hat ). I guess we might need to add more context in the original document.